velocity/src/main/java/me/confuser/banmanager/velocity/VelocityPlayer.java

```diff
@@ -0,0 +1,120 @@
+package me.confuser.banmanager.velocity;
+
+import com.velocitypowered.api.proxy.Player;
+import me.confuser.banmanager.common.CommonPlayer;
+import me.confuser.banmanager.common.CommonWorld;
+import me.confuser.banmanager.common.commands.CommonCommand;
+import me.confuser.banmanager.common.data.PlayerData;
+import me.confuser.banmanager.common.util.Message;
+import net.kyori.adventure.text.serializer.gson.GsonComponentSerializer;
+import net.kyori.text.TextComponent;
+
+import java.net.InetAddress;
+import java.util.Optional;
+import java.util.UUID;
+
+public class VelocityPlayer implements CommonPlayer {
+  private final UUID uuid;
+  private final boolean onlineMode;
+  public VelocityPlayer(Optional<Player> player, boolean onlineMode) {
```

Comment:
Why do you use an `Optional<Player>` parameter if the constructor calls `get()` on it? You might as well pass a `Player` instead.

Reply:
Thanks for highlighting this issue, it happened during the PR on my fork, didn’t review the code much, my fault had to go to bed.